http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpSamDAO.scala

```diff
@@ -3,14 +3,15 @@ package dao
 
```

Comment:
No functional changes in this file -- I just cleaned up some things, re-organized methods, broken some things into common functions.